firebase-perf/src/main/java/com/google/firebase/perf/gauges/CpuGaugeCollector.java

```diff
@@ -96,6 +97,7 @@ private CpuGaugeCollector() {
   }
 
   @VisibleForTesting
+  @Inject
```

Comment:
Also commenting here since I can't comment on the corresponding lines. As you do the exercise of DI, it would be good to do small changes / cleanups in the class structure. For example, here there is a mix of `private final`, `private`, `public`, etc variables, all mixed. 

It would be great to have "blocks" of variables so that we can instantly see the characteristics of a particular variable just through the block they belong to. It also allows us to easily see outliers that should not exist (like a public non-static variable: `cpuMetricReadings`)

Please do this throughout all these classes (no behaviour change, just cleaning up the variables section of the class)

Reply:
Great suggestion! I think it makes sense to do some small clean up on variables. Updated.

Reply:
Thanks for updating it! There's still the following that could use some sorting:

```
  @Nullable private static CpuGaugeCollector sharedInstance = null;
  @Nullable private ScheduledFuture cpuMetricCollectorJob = null;
  private final ScheduledExecutorService cpuMetricCollectorExecutor;
  private long cpuMetricCollectionRateMs = UNSET_CPU_METRIC_COLLECTION_RATE;
  private final long clockTicksPerSecond;
  private final String procFileName;
```

There's 3 separate blocks there:

- private static
- private final
- private

Reply:
Updated.